Gopkg.lock

```diff
@@ -1043,11 +1051,11 @@
 
 [[projects]]
   branch = "master"
```

Comment:
I wonder if we shouldn't use a release for this..

Reply:
Let's try to remember to adjust this to a release when one is available.

Reply:
Is [1.21.0](https://github.com/soniah/gosnmp/releases/tag/v1.21.0) sufficient (latest release from april)

Reply:
No, the upstream tcp support was just added a few weeks ago.

Reply:
Ah, right on

Reply:
Now there is a release so I used this